libafl_qemu/src/snapshot.rs

```diff
@@ -15,9 +15,11 @@ use crate::{
     GuestAddr, SYS_fstat, SYS_fstatfs, SYS_futex, SYS_getrandom, SYS_mprotect, SYS_mremap,
     SYS_munmap, SYS_pread64, SYS_read, SYS_readlinkat, SYS_statfs,
 };
+#[cfg(cpu_target = "i386")]
+use crate::{SYS_fstatat64, SYS_mmap};
```

Comment:
Why SYS_mmap here but `not` it below? Maybe split this up into fstatat and map, per arch